lib/apple_sign_in/helpers/conditions/iss_condition.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module AppleSignIn
+  module Conditions
+    class IssCondition
+      def initialize(jwt)
+        @iss = jwt['iss']
+      end
+
+      def valid?
+        @iss == AppleSignIn.config.apple_iss
```

Comment:
I think it should be that includes the iss
    `iss.include?(AppleSignIn.config.apple_iss)`

Reply:
🤔 why? if i did that then something like:
```
iss = 'hola.https://appleid.apple.com.chau' 
AppleSignIn.config.apple_iss = 'https://appleid.apple.com'
```
would be true

Reply:
Yes, it's weird but it what they say in the docs https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_rest_api/verifying_a_user `Verify that the iss field contains https://appleid.apple.com` 🤷‍♂️